cellprofiler/modules/splitormergeobjects.py

```diff
@@ -1,6 +1,16 @@
 # coding=utf-8
 
-"""
+import numpy as np
+
+import cellprofiler.image as cpi
+import cellprofiler.module as cpm
+import cellprofiler.measurement as cpmeas
+import cellprofiler.preferences as cpprefs
+import cellprofiler.setting as cps
+from cellprofiler.setting import YES, NO
+import _help
```

Comment:
It looks like only this import (`import _help`) is necessary. You can also move all the imports after the documentation string up here, too:

```python
import centrosome.cpmorphology as morph
import numpy as np
import scipy.ndimage as scind
from centrosome.filter import stretch
from scipy.sparse import coo_matrix
 
import cellprofiler.image as cpi
import cellprofiler.module as cpm
import cellprofiler.measurement as cpmeas
import cellprofiler.object as cpo
import cellprofiler.preferences as cpprefs
import cellprofiler.setting as cps
from cellprofiler.measurement import C_PARENT, C_CHILDREN, FF_CHILDREN_COUNT, FF_PARENT
from cellprofiler.modules.identify import add_object_count_measurements
from cellprofiler.modules.identify import add_object_location_measurements
from cellprofiler.modules.identify import get_object_measurement_columns
from cellprofiler.setting import YES, NO
```